src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContentAccordionSection.tsx

```diff
@@ -123,100 +144,127 @@ const ReleaseContentAccordionSection = ({
+            <EditableSectionBlocks
+              blocks={blocks}
+              isReordering={isReordering}
+              sectionId={sectionId}
+              onBlocksChange={setBlocks}
+              renderBlock={block => (
+                <ReleaseBlock
+                  block={block}
                   releaseId={release.id}
-                  onSelect={async selectedDataBlockId => {
-                    await attachDataBlock(selectedDataBlockId);
-                    toggleDataBlockForm.off();
-                  }}
-                  onCancel={toggleDataBlockForm.off}
+                  visible={open}
                 />
               )}
+              renderEditableBlock={block => (
+                <ReleaseEditableBlock
+                  allowComments
+                  allowImages
+                  block={block}
+                  sectionId={sectionId}
+                  sectionKey="content"
+                  editable={!isReordering}
+                  publicationId={release.publication.id}
+                  releaseId={release.id}
+                  visible={open}
+                />
+              )}
+            />
 
-              <ButtonGroup className="govuk-!-margin-bottom-8 dfe-justify-content--center">
-                <Button variant="secondary" onClick={addBlock}>
-                  Add text block
-                </Button>
-                {!showDataBlockForm && (
-                  <Button variant="secondary" onClick={toggleDataBlockForm.on}>
-                    Add data block
-                  </Button>
+            {editingMode === 'edit' && !isReordering && (
+              <>
+                {showDataBlockForm && (
+                  <DataBlockSelectForm
+                    id={`dataBlockSelectForm-${sectionId}`}
+                    releaseId={release.id}
+                    onSelect={async selectedDataBlockId => {
+                      await attachDataBlock(selectedDataBlockId);
+                      toggleDataBlockForm.off();
+                    }}
+                    onCancel={toggleDataBlockForm.off}
+                  />
                 )}
-              </ButtonGroup>
-            </>
-          )}
-        </>
-      )}
-    </EditableAccordionSection>
+
+                <ButtonGroup className="govuk-!-margin-bottom-8 dfe-justify-content--center">
+                  <Button variant="secondary" onClick={addBlock}>
+                    Add text block
+                  </Button>
+                  {!showDataBlockForm && (
+                    <Button
+                      variant="secondary"
+                      onClick={toggleDataBlockForm.on}
+                    >
+                      Add data block
+                    </Button>
+                  )}
+                  {!showEmbedDashboardForm && (
+                    <Button
+                      variant="secondary"
+                      onClick={toggleEmbedDashboardForm.on}
+                    >
+                      Embed dashboard
```

Comment:
Shouldn't this be 'Add embed block' or 'Add embed'?

Reply:
I think I prefer "Embed dashboard" - I know we've used embed as a noun in the codebase, but it does make sense as a verb here.